Makefile

```diff
@@ -97,7 +97,9 @@ endif
 
 ALWAYS_OFFLOAD_NODE_STATUS := false
 POD_STATUS_CAPTURE_FINALIZER ?= true
+TARGET_PLATFORM := $(shell [ `uname -m` = arm64 ] && echo linux/arm64 || echo linux/amd64)
```

Comment:
The env vars are partly categorized -- this should be placed above after `DOCKER_PUSH` or `DEV_IMAGE`